src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/DevAdventCalendarCompetition.Services.csproj

```diff
@@ -6,9 +6,14 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <DocumentationFile>C:\Users\hellg\Desktop\DevAdventCalendar\src\DevAdventCalendarCompetition\DevAdventCalendarCompetition.Services\DevAdventCalendarCompetition.Services.xml</DocumentationFile>
```

Comment:
please remove it - we don't want documentation file (and this is path to your local file ;p)